tests/test_education_crud.py

```diff
@@ -0,0 +1,163 @@
+import pry
+import json
+from tests.BaseCase import BaseCase
+from database.education import salary, zip, debt, savings, credit, percent, term, rent, principal
```

Comment:
@priyapower, one more time, what is happening on this line?

Reply:
This is importing a "temporary seeds" file that holds all the hard code. If you look under /database/education.py, you will see each of these variables

Reply:
Thank you!